pkg/providers/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -81,80 +75,12 @@ func (e EKS) eksBootstrapScript() string {
 	if (e.KubeletConfig != nil && e.KubeletConfig.MaxPods != nil) || !e.AWSENILimitedPodDensity {
 		userData.WriteString(" \\\n--use-max-pods false")
 	}
-	if args := e.kubeletExtraArgs(); len(args) > 0 {
+	if args := e.asKubeletExtraArgs(); len(args) > 0 {
```

Comment:
```suggestion
	if args := e.kubeletExtraArgs(); len(args) > 0 {
```

nit: why change this name?

Reply:
I try to avoid misleading that Options has a kubeletExtraArgs member/copy.
This kubeletExtraArgs is computed from other members, so use the "as" prefix here.